sanity-io/structured-content-2022
# Render Sponsorships section

# Render Sponsorships section

## Intent

Render the "Sponsorship Packages" table.

## Description

This PR implements table/lists under the "Sponsorship Packages" heading in the Figma sketches, following Studio changes in #120. This PR also includes a bit of refactoring to pull out and reuse some components already defined by _Tickets.tsx_.

## Testing this PR

1. Go to [/sponsorship-information](https://structured-content-2022-web-git-feature-sc-14927sponsors-08bb9f.sanity.build/sponsorship-information)
2. Verify that the table under "Sponsorship Packages" renders all the fields specified by the [Figma sketch](https://www.figma.com/file/XjAvJKsLpxYJSc8QDWK81j/Sanity-SCC-Website-(Shared)?node-id=1758%3A8699) and looks good in all specified viewports
3. Go to [/registration-info](https://structured-content-2022-web-git-feature-sc-14927sponsors-08bb9f.sanity.build/registration-info)
4. Verify that the fields that still render look OK and that the _/Tickets_ component refactorings in this PR don't break the component
    * Note that #119 introduces new fields to Tickets, which likely is why some fields aren't rendered locally/in Preview Deploys
5. (Push desired fixes directly to this PR if you want to)
